File: app/src/main/java/com/kontranik/koreader/parser/fb2reader/model/FB2Scheme.java
package com.kontranik.koreader.parser.fb2reader.model;

import java.util.ArrayList;
import java.util.List;

public class FB2Scheme {

    public FB2Description description = new FB2Description();
	public List<FB2Section> sections = new ArrayList<>();
	public BinaryData cover;

	public FB2Section getSection(Integer orderId) {
		return sections.get(orderId);
	}

	public FB2Section getSection(String attrId) {
		for (FB2Section fb2Section : sections) {
			if ( fb2Section.id != null && fb2Section.id.equals(attrId) ) {
					return fb2Section;
			}
		}
		return null;
	}

	/**
	 * @return the description
	 */
	public FB2Description getDescription() {
		return description;
	}

	/**
	 * @param description the description to set
	 */
	public void setDescription(FB2Description description) {
		this.description = description;
	}

	/**
	 * @return the sections
	 */
	public List<FB2Section> getSections() {
		return sections;
	}

	/**
	 * @param sections the sections to set
	 */
	public void setSections(List<FB2Section> sections) {
		this.sections = sections;
	}

}

File: app/src/main/java/com/kontranik/koreader/parser/fb2reader/parser/model/FB2Scheme.kt
package com.kontranik.koreader.parser.fb2reader.parser.model

class FB2Scheme {

    var description = FB2Description()
    var sections: MutableList<FB2Section> = mutableListOf()
    var cover: BinaryData? = null
    var path: String? = null

    fun getSection(orderId: Int?): FB2Section {
        return sections[orderId!!]
    }

    fun getSection(attrId: String): FB2Section? {
        for (fb2Section in sections) {
            if (fb2Section.id != null && fb2Section.id == attrId) {
                return fb2Section
            }
        }
        return null
    }
}